plugins/modules/swim_workflow_manager.py

```diff
@@ -914,7 +1102,7 @@ def get_have(self):
             # check if given site exists, store siteid
             # if not then use global site
             site_name = tagging_details.get("site_name")
-            if site_name and site_name != "Global":
+            if site_name:
```

Comment:
Why you remove this check ?
site_name != "Global":

Reply:
The removal of the site_name != "Global" check was unintentional. I'll ensure it's reintroduced into the code to maintain the correct logic and functionality.